MCprep_addon/materials/material_manager.py

```diff
@@ -285,90 +289,105 @@ class MCPREP_OT_combine_images(bpy.types.Operator):
 	def execute(self, context):
 		removeold = True
 
-		if self.selection_only is True and len(context.selected_objects) == 0:
+		if bpy.app.version < (2, 78):
```

Comment:
We already support only Blender 2.8+ so we can just remove this block (appreciate trying to maintain the functionality though)